lazer/contracts/sui/sdk/js/eslint.config.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line n/no-unpublished-import
```

Comment:
This wouldn't be necessary if you excluded `eslint.config.js` from the `files` array in the `package.json` -- basically ensure that `files` just contains `lib/**/*` since everything else is dev tooling